ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/BitmovinPlayerPlugin.swift

```diff
@@ -40,7 +38,13 @@ extension BitmovinPlayerPlugin: ZPPlayerProtocol {
 
     public static func pluggablePlayerInit(playableItems items: [ZPPlayable]?, configurationJSON: NSDictionary?) -> ZPPlayerProtocol? {
 
-        guard let videos = items else { return nil }
+        var videos: [ZPPlayable] = items ?? [ZPPlayable]()
+
+        if let d = configurationJSON,
+            let testUrl = d["test_video_url"] as? String {
```

Comment:
You should check also that the string is not empty. We don't want to do this if the string is "".